test.ts

```diff
@@ -413,6 +421,138 @@ Deno.test({
     },
 });
 
+Deno.test({
+    name: "Invitation",
+    // ignore: true,
+    fn: async (t) => {
+        const relay = await run({
+            admin: test_ctx.publicKey,
+            auth_required: true,
+            default_policy: {
+                allowed_kinds: "none", // EventV2 is not currently restricted by Policy
+            },
+        });
+        if (relay instanceof Error) fail(relay.message);
+        const invitor = InMemoryAccountContext.Generate();
+        const acceptor = InMemoryAccountContext.Generate();
+        const invite: InviteToSpace = await sign_event_v2(invitor.privateKey, {
```

Comment:
Do you think that we need a similar interface such as
```
prepareInvitation(invitor)
```
?

Reply:
Of course